examples/elasticache/v1beta2/replicationgroup.yaml

```diff
@@ -14,13 +68,25 @@ metadata:
 spec:
   forProvider:
     automaticFailoverEnabled: true
+    atRestEncryptionEnabled: true
+    autoGenerateAuthToken: true
+    authTokenSecretRef:
+      name: redis-auth-token
+      namespace: crossplane-system
```

Comment:
The namespace `crossplane-system` is not available by default. Let's change this to `upbound-system`.